backend/src/main/java/com/benine/backend/video/Stream.java

```diff
@@ -0,0 +1,57 @@
+package com.benine.backend.video;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.net.URLConnection;
+
+/**
+ * Defines a Stream connection.
+ */
+public class Stream {
+
+  private URLConnection connection;
+  private InputStream inputstream;
+
+  /**
+   * Constructor for a new stream object.
+   * @param streamurl The url to get the stream from.
+   * @throws IOException If an exception occurs while creating the stream,
+   * rendering the stream useless.
```

Comment:
This gives a small checkstyle warning, rendering should be right under IOException.